docs/notebooks/introduction.pct.py

```diff
@@ -243,3 +245,79 @@
     num_init=len(dataset.query_points),
     idx_best=arg_min_idx,
 )
+
+# %% [markdown]
+# # Try an alternative acquisition function
+
+# %% [markdown]
+# By default, Trieste uses Expected Improvement (EI) as its acqusition function when performing Bayesian optimization. However, many alternative acqusition functions have been developed. One such alternative is Max-value Entropy Search (MES), which approximates the distribution of current estimate of the global minimum and tries to decrease its entropy with each optimization step.
+
+# %% [markdown]
+# We plot these two acquisition functions across our search space. Areas with high acquisition function scores (i.e bright regions) are those rated as promising locations for the next evaluation of our objective function. We see that EI wishes to continue exploring the search space, whereas MES wants to focus resources on evaluating a specific region.
+
+# %%
+mes_acq_function = trieste.acquisition.MaxValueEntropySearch(search_space,num_samples = 5, grid_size = 5000).using(OBJECTIVE).prepare_acquisition_function(initial_data, model)
+ei_acq_function = trieste.acquisition.ExpectedImprovement().using(OBJECTIVE).prepare_acquisition_function(initial_data, model)
+
+fig, ax = plot_function_2d(mes_acq_function, mins, maxs, grid_density=40, contour=True)
+plot_bo_points(
+    dataset.query_points.numpy(),
+    ax=ax[0, 0],
+    num_init=len(dataset.query_points),
+    idx_best=arg_min_idx,
+)
+fig.suptitle("Max-value Entropy Search Acquisition Function")
+fig, ax = plot_function_2d(ei_acq_function, mins, maxs, grid_density=40, contour=True)
+plot_bo_points(
+    dataset.query_points.numpy(),
+    ax=ax[0, 0],
+    num_init=len(dataset.query_points),
+    idx_best=arg_min_idx,
+)
+fig.suptitle("Expected Improvement Acquisition Function")
+
+# %% [markdown]
+# To compare the performance of the optimization achieved by these two different acquisition functions, we re-run the above BO loop using MES. 
+
+# %% [markdown]
+# We re-initialize the model and define a new acquisiton rule.
+
+# %%
+gpr = gpflow.models.GPR(astuple(initial_data[OBJECTIVE]), kernel, noise_variance=1e-5)
```

Comment:
this will use the kernel from before, which has been trained. Do we not need a new kernel? A function would be good for that
```python
def build_model(data):
    variance = tf.math.reduce_variance(data.observations)
    kernel = gpflow.kernels.Matern52(variance=variance, lengthscales=0.2 * np.ones(2,))
    gpr = gpflow.models.GPR(astuple(data), kernel, noise_variance=1e-5)
    set_trainable(gpr.likelihood, False)
    
    return trieste.models.create_model_interface({
        "model": gpr,
        "optimizer": gpflow.optimizers.Scipy(),
        "optimizer_args": {"options": dict(maxiter=100)},
    })
```